helper/metricsutil/wrapped_metrics.go

```diff
@@ -88,8 +88,9 @@ func (m *ClusterMetricSink) MeasureSinceWithLabels(key []string, start time.Time
 
 // BlackholeSink is a default suitable for use in unit tests.
 func BlackholeSink() *ClusterMetricSink {
-	sink, _ := metrics.New(metrics.DefaultConfig(""),
-		&metrics.BlackholeSink{})
+	conf := metrics.DefaultConfig("")
+	conf.EnableRuntimeMetrics = false
```

Comment:
Was this intentionally disabled?

Reply:
Yes, to avoid it creating a goroutine that interferes with the leak test.  I believe BlackholeSink is only used in tests.